manifests/init.pp

```diff
@@ -131,6 +138,8 @@
   Stdlib::Absolutepath $duply_archive_install_dir = $duplicity::params::duply_archive_install_dir,
   Optional[String] $duply_version = undef,
   Stdlib::Absolutepath $duply_archive_executable = $duplicity::params::duply_archive_executable,
+  String $duply_log_output = $duplicity::params::duply_log_output,
```

Comment:
Given this parameter requires additional logic inside the module, this is a great fit for an `Enum`, what do you think?

```suggestion
  Enum['file', 'logger'] $duply_log_output = $duplicity::params::duply_log_output,
```

Reply:
Yes, for sure, it's better to use Enum here.